airflow/jobs/scheduler_job_runner.py

```diff
@@ -1277,6 +1277,10 @@ def _create_dag_runs_dataset_triggered(
                 ]
                 if previous_dag_run:
                     dataset_event_filters.append(DatasetEvent.timestamp > previous_dag_run.execution_date)
+                else:
+                    dataset_event_filters.append(
+                        DatasetEvent.timestamp >= DagScheduleDatasetReference.created_at
+                    )
```

Comment:
Hmm just wondering if the difference between `>=` and `>` is something we should worry about.

Reply:
nope, moved it back to `>`